content/javascript/concepts/bitwise-operators/bitwise-operators.md

```diff
@@ -37,6 +37,15 @@ console.log(19 & 7); // Output: 3
 // 00011 =  3
 ```
 
+## Codebyte Example
```

Comment:
Change these headings(All codebyte headings) to H3 as their parent heading is H3, this will keep the heading hierarchy maintained.

Reply:
Thanks for your feedback. New updates for the H3 elements have now been made.